### PR TITLE
refactor(autoware_pointcloud_preprocessor): rework pickup based voxel grid downsample filter parameters

### PR DESCRIPTION
**Pull request by vividf**
## Description
This PR includes the following changes
1. Remove the default parameters for the pickup based voxel grid downsample filter node. 
2. Add parameter, launch file, and parameters schema for the node. 

A big thank you to @Ariiees for the PRs (https://github.com/autowarefoundation/autoware.universe/pull/8297 and https://github.com/autowarefoundation/autoware.universe/pull/7422), where we worked together on the entire pointcloud preprocessor parameters, launch file, and schema.
## Related links

**Parent Issue:**

- Link

## How was this PR tested?
`ros2 launch autoware_pointcloud_preprocessor pickup_based_voxel_grid_downsample_filter_node.launch.xml`
## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

File: sensing/autoware_pointcloud_preprocessor/schema/pickup_based_voxel_grid_downsample_filter_node.schema.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  "definitions": {
+    "pickup_based_voxel_grid_downsample_filter": {
+      "type": "object",
+      "properties": {
+        "voxel_size_x": {
+          "type": "number",
+          "description": "voxel size along x-axis [m]",
+          "default": "1.0"
+        },
+        "voxel_size_y": {
```
Comment (knzo25) [inline sensing/autoware_pointcloud_preprocessor/schema/pickup_based_voxel_grid_downsample_filter_node.schema.json:14]: Can you add limitations to these values?
(applies to all other related PRs)

**Review by knzo25 — CHANGES_REQUESTED** [review]
wrote a schema related comment

File: sensing/autoware_pointcloud_preprocessor/schema/pickup_based_voxel_grid_downsample_filter_node.schema.json (hunk 1 above)
Comment (vividf) [reply sensing/autoware_pointcloud_preprocessor/schema/pickup_based_voxel_grid_downsample_filter_node.schema.json:15]: Thanks, fixed them in https://github.com/autowarefoundation/autoware.universe/pull/8481/commits/40feff625de91cc41ea5406aacdce703ce6223fe. 

* I also update boundary in other PRs

**Comment by vividf** [comment]
@knzo25 kindly ping

File: sensing/autoware_pointcloud_preprocessor/schema/pickup_based_voxel_grid_downsample_filter_node.schema.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Pickup Based Voxel Grid Downsample Filter Node",
+  "type": "object",
+  "definitions": {
+    "pickup_based_voxel_grid_downsample_filter": {
+      "type": "object",
+      "properties": {
+        "voxel_size_x": {
+          "type": "number",
+          "description": "voxel size along x-axis [m]",
```
Comment (knzo25) [inline sensing/autoware_pointcloud_preprocessor/schema/pickup_based_voxel_grid_downsample_filter_node.schema.json:11]:
```suggestion
          "description": "voxel size along the x-axis [m]",
```

Comment (knzo25) [reply sensing/autoware_pointcloud_preprocessor/schema/pickup_based_voxel_grid_downsample_filter_node.schema.json:11]: This also applies to the other axes. I think there are more occurrences of this on other of your PRs

**Review by knzo25 — APPROVED** [review]
LGTM

**Pull request merged**